qdrant-landing/themes/qdrant-2024/layouts/partials/vectors.html

```diff
@@ -1,7 +1,18 @@
-<section class="vectors">
-  <div class="container">
+{{ $defaultClass := "vectors" }}
+{{ $cssClass := "" }}
+{{ if .Scratch.Get "cssClass" }}
+  {{ $cssClass = .Scratch.Get
+    "cssClass"
+  }}
+{{ end }}
+
+
+<section class="{{ printf "%s %s" $defaultClass $cssClass }}">
+  <div class="vectors__header">
```

Comment:
If you will add .container -> .row -> .col-12 like on the `/qdrant-vector-database` page it solves the next issue (screenshot). And then you can get rid of media queries in the corresponding styles.

![Screenshot 2024-03-27 at 12 30 36](https://github.com/qdrant/landing_page/assets/7085263/a3ac1610-7486-4542-a9dc-4ee6333f08e1)